opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -66,18 +70,23 @@ class SimpleExportSpanProcessor(SpanProcessor):
     passes ended spans directly to the configured `SpanExporter`.
     """
 
-    def __init__(self, span_exporter: SpanExporter):
+    def __init__(
+        self, span_exporter: SpanExporter, timeout: int = None,
```

Comment:
It's ok to have no timeout as the default timeout?

Reply:
I'd vote for a default timeout, maybe 60 sec that could return an warning to the user that something's gone wrong.

Reply:
Okay! 60s sounds like a reasonable time to wait :)